kuma/javascript/src/article.jsx

```diff
@@ -144,20 +144,19 @@ export default function Article({ document }: DocumentProps) {
         }
     }, [document, userData]);
 
+    const isArchive =
+        document.slug === 'Archive' || document.slug.startsWith('Archive/');
+
     return (
         /*
          * The "text-content" class and "wikiArticle" id are required
          * because our stylesheets expect them and formatting isn't quite
          * right without them.
          */
-        <div
-            id="content"
-            ref={article}
-            className="text-content"
-            css={styles.article}
-        >
+        <div id="content" ref={article} css={styles.article}>
             <article
                 id="wikiArticle"
+                className={isArchive ? 'archive-content' : null}
```

Comment:
Given that the set of styles is pretty small for archived pages, how about just copying those styles here with emotion rather than messing around with the existing stylesheets?

Reply:
That way  you could fix this bug without having to change anything for the wiki site, which would be safer.

Reply:
I am a little hesitant to move CSS to JS to quickly. I would prefer to keep this in Sass.

Reply:
This is a big topic that mixes personal preferences together with the fact that we still have the wiki. 
By doing it all from 1 single .scss file we avoid DRY. Doing it all in SCSS means you can reuse variables for things like colors and media query tricks. 
Pragmatically I think emotion would be a better home because it should be easy peasy to tie it into the business logic conditional. 

How much other CSS things have been copy-n-pasted over from SCSS files into emotion in JS? 

Also, god forbid we decide to change the color from orange to maroon one day and we forget our DRY violation of July 2019. Then, how big a deal is it that it looks different in the wiki compared to the read-only site. Do the 1% care?